test/schema_generators/generators_test.cljc

```diff
@@ -80,3 +81,20 @@
 (defspec can-mix-wildcard-keys-with-specific-keys 50
   (properties/for-all [m (generators/generator Issue16RegressionSchema)]
     (is (number? (:x m)))))
+
+(def FailRetries
+  (s/conditional
+    int? s/Str))
+
+(deftest maximum-retries-test
+  (is (thrown-with-msg?
```

Comment:
These tests feel pretty redundant.   Checking the error message is also OK but it might be nicer to create a function that e.g. accepts the nth proposal and check that generation works if max-retries is >n but not <n.

Reply:
That seems like a good test, but I have no knowledge on how to access the number of proposals during a generation process

Reply:
Changes on https://github.com/plumatic/schema-generators/pull/29/commits/7043572018c08c519a1fa712d9f3ef7834acb3b6 and https://github.com/plumatic/schema-generators/pull/29/commits/3bc12dcfb35437788737f4b5dbf311d805fd4e2f